### PR TITLE
Add java auto instrumentation steps

### PR DESCRIPTION
**Pull request by shavidissa**
Add the Java auto instrumentation steps from https://tanzu.vmware.com/content/blog/opentelemetry-metrics-trace-data-aria-operations-for-applications.
Restructure folders, 
name change: Python

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,72 +34,32 @@ Follow these steps:
...
     * **Metrics data**:
         <br/>Explore the metrics data you sent with charts and dashboards.
         * Try out the [Dashboards and Charts tutorial](https://docs.wavefront.com/tutorial_dashboards.html), or watch the video on that page to get started.
-        * Create [dashboards](https://docs.wavefront.com/ui_dashboards.html) and [charts](https://docs.wavefront.com/ui_charts.html) for the data you sent to Tanzu Observability. 
+        * Create [dashboards](https://docs.wavefront.com/ui_dashboards.html) and [charts](https://docs.wavefront.com/ui_charts.html) using the data you sent to our service. 
           <br/>You need to have the required permissions to do these tasks.
 
 
-
-## Send Data Using the OpenTelemetry Collector and the Wavefront Proxy
```
Comment (shavidissa) [inline README.md:52]: @keep94 @oppegard 
Deleted this section as it it no longer relevant.

File: java-example/auto-instrumentation/README.MD [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Auto Instrumenting Java Apps with OpenTelemetry
```
Comment (shavidissa) [inline java-example/auto-instrumentation/README.MD:1]: @keep94 @oppegard 
This is the new content from the blog. Did some minor formatting change. I will test it out too within this week

File: DOTNET-example/README.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
 # Instrumenting .NET Apps with OpenTelemetry
 
-## Auto-Instrumentation
+## Instrument Using a Library
```
Comment (shavidissa) [inline DOTNET-example/README.md:3]: @oppegard : Changed this as discussed at the meeting.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,21 @@
-# Sending Data to Tanzu Observability by Wavefront
+# Sending Data to Our Service
 
-If you use OpenTelemetry, you can configure the application to send traces or metrics to Tanzu Observability using the Wavefront Proxy or the OpenTelemetry Collector.
+If you use OpenTelemetry, you can configure the application to send traces or metrics to VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) using the Wavefront proxy or the OpenTelemetry Collector.
 
-## Directly Send Data Using the Wavefront Proxy - (Recommended)
+Send trace data or metrics data to our service using only the Wavefront proxy:
```
Comment (keep94) [inline README.md:5]: Not sure if WF proxy is still called WF proxy.

**Comment by deosu** [comment]
@shavidissa I have fixed the GitHub actions also I have merged previous PRs to master and your branch is now up to date with the master branch. All the checks are green now. Let me know if you need anything. 
cc: @keep94

**Comment by shavidissa** [comment]
Thank you so much, @deosu!

File: README.md (hunk 4 above)
Comment (shavidissa) [reply README.md:5]: We still call it Wavefront proxy :)

File: DOTNET-example/README.md [hunk 5]
```diff
@@ -27,13 +26,13 @@ will eventually export the trace data to the Tanzu Observability UI.
...
-    * `{INSTANCE_NAME}` with the Tanzu Observability instance (for example, https://longboard.wavefront.com).
-    * `{TOKEN}` with a Tanzu Observability API token linked to an account with Proxy permission.
+    * `{INSTANCE_NAME}` with the product instance (for example, https://longboard.wavefront.com).
+    * `{TOKEN}` with the API token linked to an account with Proxy permission.
       See [Generating and an API Token](https://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
     
     See [Install a Proxy](http://docs.wavefront.com/proxies_installing.html#install-a-proxy) to find other options for installing the proxy on your environment.
     
-* Set up an OpenTelemetry Collector for Tanzu Observability:
+* Set up an OpenTelemetry Collector:
```
[the remark this answers is not in the document]
Comment (shavidissa) [reply DOTNET-example/README.md:35]: Got it! This is the .net example.. did we have a blog post for .net? 
Yeah..all our tutorials use the collector for auto instrumentation steps.. Juhi said we will update the content in the next few iterations :)

[the remark this answers is not in the document]
Comment (shavidissa) [reply DOTNET-example/README.md:35]: Does that sound ok? What do you think, @keep94 ?

File: DOTNET-example/README.md (hunk 3 above)
Comment (shavidissa) [reply DOTNET-example/README.md:3]: Hi Travis,
Thank you! Let's add the auto instrumentation steps later on as discussed at the meeting. Will leave this content as we discussed.

**Review by keep94 — APPROVED** [review]
Thx for the great work Shavi.

**Comment by shavidissa** [comment]
Thank you so much!

**Pull request merged**